src/test/java/com/myhome/services/HouseMemberDocumentServiceTestAdditional.java

```diff
@@ -0,0 +1,179 @@
+package com.myhome.services;
+
+import com.myhome.domain.HouseMember;
+import com.myhome.domain.HouseMemberDocument;
+import com.myhome.repositories.HouseMemberDocumentRepository;
+import com.myhome.repositories.HouseMemberRepository;
+import com.myhome.services.springdatajpa.HouseMemberDocumentSDJpaService;
+import org.junit.Ignore;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.mock.web.MockMultipartFile;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+@SpringBootTest
```

Comment:
Isn't this a unit test. Why is entire context necessary? I see all the dependencies being mocked.
In that case, can this annotation be removed.

Also, it is appropriate if this file is moved to `unit` folder.

Reply:
Fixed